notification/src/models/notificationConfig.ts

```diff
@@ -0,0 +1,65 @@
+import { Entity,  Column,  PrimaryGeneratedColumn } from 'typeorm'
+
+export enum CONFIG_TYPE{
+  WEBHOOK = "webhook",
+  SLACK = "slack",
+  FCM = "fcm"
+}
+
+
+export class NotificationConfig{
```

Comment:
Can two instances of notificationConfig have the same id because they are instances of different subtypes? I do not know if there is an easy way to fix this, but this seems inconsistent to me.

Reply:
With the current database architecture this cannot be achieved and requires major refactoring on the service side to implement it in a proper way. A solution i can think of is that we can genereate a composed identifier that consists of the type and the autogenerated id (e.g. `SLACK_1` or `SLACK_1` cast as an number)